active/fellowship.md

```diff
@@ -2,30 +2,62 @@
+  - As with the CoC-WG, the Fellowing-WG has the power to unilaterally suspend Fellows, with pay, pending an investigation. This power should only be used in egregious circumstances, where waiting on investigation and/or board action might plausibly be harmful.
+- Designating one member of the committee to act as the line manager for active fellows.
+  - The Board must approve this designation.
+  - Defining what it means to be a manager is out of scope for this document, but it's all the typical stuff: one-on-ones, feedback, coaching, and performance management, if necessary.
+  - It's expected that this person will have previous engineering management experience.
+  - This person also approves expenses for Fellows, subject to the [budget](#budget), see below.
+  - Perform bi-annual performance reviews for the fellows, and reports to the board.
+    - This can be super informal, short and sweet is fine. This is intended to be a backstop against the unlikely event of performance issues with a fellow, not any sort of stack ranking etc.
+- Acting as an escalation point if a Fellow has a disagreement with their designated manager. That is, the escalation path for Fellows is: designated manager -> full Fellowship WG -> Board.
+- Maintaining the formal job description and outline of duties for Fellows. Those changes must be reported to the Board who may veto them by board vote.
+- Gathering and maintaining statistics on the impact and efficacy of the Fellow program (for use in fundraising, reports to sponsors, etc). See [reporting](#reporting) for a short discussion of this report.
+- TKTK am I missing anything?
 
 ## Initial membership
 
-- Chair: Brian Moloney
-- Board liaison: Chaim Kirby
+Upon adoption of this new charter, the Board will issue an open call for members, recruit, and select membership. We'll follow the guidelines outlined below under [future membership](#future-membership) in selecting this initial group./
+
+- Chair: TBD
+- Board liaison: TBD
+- Designated manager: TBD
 - Other members:
-  - Andrew Godwin
-  - Frank Wiles
+  - TBD
 
 ## Future membership
 
-Board-managed: the DSF Board appoints members of this WG.
+Board-managed: the DSF Board appoints members of this WG, including specific roles (chair, Fellow manager).
+
+The Board will select membership with some high-level goals in mind:
+
+- **Diversity**: the WG membership should be diverse, as representative of the global Django community as possible within the confines of a small team.
+- **Management experience**: WG members, _particularly_ the designated manager, should have some engineering management experience, or other relevant experience (i.e. experience with fellowship programs at other NPOs, etc). This isn't a hard
```

Comment:
Note: This line just... ends. I suspect it wants to at least have ` requirement`.